tests/cypress/support/commands.ts

```diff
@@ -152,27 +155,30 @@ Cypress.Commands.add('deployToClusterOrClusterGroup', (deployToTarget) => {
 
 // 3 dots menu selection
 Cypress.Commands.add('open3dotsMenu', (name, selection, checkNotInMenu=false) => {
+  // Let cy.filterInSearchBox() operation finish and wait before opening Edit Menu.
+  cy.wait(500);
   // Open 3 dots button
   cy.contains('tr.main-row', name).should('exist').within(() => {
-    cy.get('.icon.icon-actions').click({ force: true });
-    cy.wait(250)
+    cy.get('.icon.icon-actions', { timeout: 500 }).click({ force: true });
+    // cy.wait(250)
   });
 
   if (checkNotInMenu === true) {
-    cy.get('.list-unstyled.menu > li').each(($el) => {
+    cy.get('.list-unstyled.menu > li, [role="menuitem"]').each(($el) => {
         if ($el.text() != selection) {
-        cy.log(`Cannot perform action with specified value "${selection}" since it is not present. Current Menu is: "${$el.text()}"`);
-        cy.get('ul.list-unstyled.menu').contains(selection).should('not.exist')
-      }        
+          cy.log(`Cannot perform action with specified value "${selection}" since it is not present. Current Menu is: "${$el.text()}"`);
+          cy.get('ul.list-unstyled.menu, [role="menuitem"]')
+            .contains(selection).should('not.exist');
+        }
     });
     // Close 3 dots button menu
-    cy.get('.background').should('exist').click({ force: true });
+    cy.get('[class="main-layout"]').should('exist').click({ force: true });
   }
   
   else if (selection) {
     // Open edit config and select option
-    cy.get('.list-unstyled.menu > li > span', { timeout: 15000 }).contains(selection).should('be.visible');
-    cy.get('.list-unstyled.menu > li > span', { timeout: 15000 }).contains(selection).click({ force: true });
+    cy.get('.list-unstyled.menu > li > span, div.dropdownTarget', { timeout: 15000 }).contains(selection).should('be.visible');
```

Comment:
mmm, does this work with the comma after `span` : `span, div.dropdownTarget'`? 
I have not seen yet a locator with this type of construction

Reply:
Yes, it is added after `span` separated by comma.

Reply:
I will add it.

Reply:
I checked that running locally if we wanted to use multiple locators then we have to use comma separated locators with args.